Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/JointDriveController.cs

```diff
@@ -122,7 +122,7 @@ public void SetupBodyPart(Transform t)
                 startingPos = t.position,
                 startingRot = t.rotation
             };
-            bp.rb.maxAngularVelocity = 100;
```

Comment:
Can we make this value a constant?

Reply:
Something like : 
```csharp
const float k_MaxAngularVelocity = 50.0;
...
bp.rb.maxAngularVelocity = k_MaxAngularVelocity;
```

Reply:
done. I changed it to your example. We have other JointDriveController changes to make but i think those should be in a different PR